dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/BitstreamRestRepository.java

```diff
@@ -126,9 +126,9 @@ protected void delete(Context context, UUID id) throws AuthorizeException {
             if (bit == null) {
                 throw new ResourceNotFoundException("The bitstream with uuid " + id + " could not be found");
             }
-            if (bit.isDeleted()) {
-                throw new ResourceNotFoundException("The bitstream with uuid " + id + " was already deleted");
-            }
+            // if (bit.isDeleted()) {
```

Comment:
this allows deleting of stuck bitstreams. (Deletion of an allready deleted item should be harmless)